src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java

```diff
@@ -1238,14 +1238,6 @@ public List<Long> selectFilesWithMissingOriginalSizes() {
     }
     
 
-    /**
-     * Check that a identifier entered by the user is unique (not currently used
-     * for any other study in this Dataverse Network). Also check for duplicate
-     * in the remote PID service if needed
-     * @param datafileId
-     * @param storageLocation
-     * @return  {@code true} iff the global identifier is unique.
-     */
```

Comment:
Not related to the PR - this just doesn't fit the method below which doesn't involve PIDs